practice3.rb

```diff
@@ -2,4 +2,4 @@
     first_number = gets.chomp.to_i
 print "Enter the second number:"
     second_number = gets.chomp.to_i
-puts "The sum of the numbers is: #{first_number + second_number}"
+puts "The sum of the numbers is: #{first_number + second_number}"
```

Comment:
Remove the `practice3.rb` file and just edit `practice2.rb` to use interpolation and remove sum.
Do `git reset --hard b6f87f8c44ff6d8f6d905474d77d5d034efe6da3` para ma-rollback yung changes mo.